### PR TITLE
feat: centered header row title

### PR DESCRIPTION
**Pull request by cartcrom**
Part of [WEB-2712](https://uniswaplabs.atlassian.net/browse/WEB-2712), center's the title on the header component. Also simplifies the wrapping of the component and removes redundant properties.

Also removed unnused "recent transactions" page so that I could simplify the header further

new:
image

old:
image

[WEB-2712]: https://uniswaplabs.atlassian.net/browse/WEB-2712?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Dialog.tsx [hunk 1]
```diff
@@ -67,37 +64,46 @@ export function Provider({ value, children }: ProviderProps) {
 const OnCloseContext = createContext<(() => void) | undefined>(undefined)
 
 const HeaderRow = styled(Row)`
```
Comment (just-toby) [inline src/components/Dialog.tsx:66]: should this be

```suggestion
const HeaderRow = styled(Row)<{iconSize?: number}>`
```

? i am new to styled components, so i'm not sure if we should always use the prop types

File: src/components/Dialog.tsx [hunk 2]
```diff
@@ -67,37 +64,46 @@ export function Provider({ value, children }: ProviderProps) {
...
+  :hover {
+    cursor: pointer;
+    opacity: 0.6;
+  }
+`
+
+const Title = styled.div`
+  left: 50%;
+  position: absolute;
+  top: 50%;
```
Comment (just-toby) [inline src/components/Dialog.tsx:90]: i could see why we want absolute positioning on the horizontal axis, but should we stick with flex / `align-items: center` for vertical spacing here?

Comment (just-toby) [reply src/components/Dialog.tsx:90]: nit / non blocking

**Review by just-toby — APPROVED** [review]

File: src/components/Dialog.tsx (hunk 2 above)
Comment (cartcrom) [reply src/components/Dialog.tsx:90]: excellent point, will do ty

**Pull request merged**